consensus/src/extension/election.rs

```diff
@@ -356,4 +357,31 @@ mod test {
             }
         }
     }
+
+    #[test]
+    #[ignore]
+    fn given_minimal_dag_with_orphaned_node_when_electing_then_orphaned_node_is_not_head() {
```

Comment:
Btw, doesn't this test pass, even in the current version? I kinda would've suspected it should, although I haven't thought about this too deeply.

Reply:
It does pass in a situation there are direct parents only in the dag. We'll see what happens after changes to parent voting code.